lib/std/wordwrap.nim

```diff
@@ -113,7 +113,6 @@ tnaetdriaoenvlcyfglwckßqfgvwkßqgfvlwkßqfgvlwckßqvlwkgfUIαοιαοιαχολ
 This string is wrapped to 60 characters. If we call
 wrapwords on it it will be re-wrapped to 80 characters.
 """
-  const input60Res = """
-This string is wrapped to 60 characters. If we call wrapwords on it it will be
-re-wrapped to 80 characters.""".strip()
+  const input60Res = """This string is wrapped to 60 characters. If we call wrapwords on it it will be
```

Comment:
space characters in 1st line are not treated as part of the multiline litteral, so that this also works:

```nim
const input60Res = """
This string is wrapped ...."""
```

Reply:
Thanks - I was sure I'd read something like that but then failed to see it _immediately_ in the documentation (not denying it's there!)